pyproject.toml

```diff
@@ -1,36 +1,34 @@
-[tool.poetry]
+[project]
 name = "gobble"
 version = "1.0.0"
 description = "Process MBTA events into a format that can be consumed by the Data Dashboard"
 authors = ["TransitMatters Labs <labs@transitmatters.org>"]
+requires-python = ">=3.12,<3.13"
 license = "MIT"
 readme = "README.md"
 package-mode = false
+dependencies = [
+    "pandas==2.2.3",
+    "sseclient-py==1.8.0",
+    "requests==2.32.5",
+    "boto3~=1.38.30",
+    "ddtrace~=3.13.0",
+    "python-json-logger~=2.0.7",
+    "coverage~=7.6.10",
+    "debugpy~=1.8.14",
+]
 
-[tool.poetry.dependencies]
-python = "~3.12"
-pandas = "2.2.3"
-sseclient-py = "1.8.0"
-requests = "2.32.5"
-boto3 = "^1.38.30"
-ddtrace = "^3.13.0"
-python-json-logger = "^2.0.7"
-coverage = "^7.6.10"
-debugpy = "^1.8.14"
+[dependency-groups]
+dev = [
+    "chalice==1.32.0",
+    "ruff>=0.14.3",
+    "pytest~=8.3.5",
```

Comment:
Any reason why we downgraded from "~8.4.2"?

Reply:
Not sure how that happened tbh - good catch, thanks!